pyproject.toml

```diff
@@ -77,7 +78,12 @@ docs = [
 ]
 
 [project.urls]
-repository = "https://github.com/DKISTDC/dkist"
+Homepage = "https://github.com/DKISTDC/dkist"
+"Source Code" = "https://github.com/DKISTDC/dkist"
+Download = "https://pypi.org/project/dkist"
+Documentation = "https://docs.dkist.nso.edu/projects/python-tools/en/stable/"
+Changelog = "https://docs.dkist.nso.edu/projects/python-tools/en/stable/whatsnew/changelog.html"
+"Issue Tracker" = "https://github.com/DKISTDC/dkist/issues"
```

Comment:
This info is all duplicated in .cruft.json, is that deliberate?

Reply:
`.cruft.json` is the source and this is where they get used.